demisto_sdk/commands/content_graph/parsers/integration.py

```diff
@@ -50,6 +56,11 @@ def __init__(
         self.connect_to_commands()
         self.connect_to_dependencies()
         self.connect_to_tests()
+        self.structure_errors = self.validate_structure()
```

Comment:
can we make this automatically run in all objects, instead of writing it in each?